src/main.ts

```diff
@@ -2,19 +2,17 @@ import Vue from "vue";
 import { BootstrapVue, IconsPlugin } from "bootstrap-vue";
 
 import App from "./App.vue";
-// import router from "./router";
+import router from "./router";
 
 import "bootstrap/dist/css/bootstrap.css";
 import "bootstrap-vue/dist/bootstrap-vue.css";
 
 import "./assets/main.css";
 
-// Make BootstrapVue available throughout your project
 Vue.use(BootstrapVue);
```

Comment:
better to avoid global import for all components. You can just import it in the component
```js
    import {BFormInput} from 'bootstrap-vue';
```